plugins/action_review_visibility/plugin.json

```diff
@@ -3,7 +3,8 @@
   "code": "updateReviewVisibility",
```

Comment:
Should the folder be renamed to `action_update_review_visibility`?

Reply:
Yeah sure.